.github/ISSUE_TEMPLATE/support_request.md

```diff
@@ -0,0 +1,9 @@
+---
+name: ⛔ Support request
+about: See https://github.com/banzaicloud/pipeline/blob/master/.github/SUPPORT.md for support channels
```

Comment:
We could probably use an URL shortener for this? Even a self-hosted one?

Reply:
will this be clickable on the ui?